roles/jenkins_builder/tasks/jenkins_credential.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Placing ssh public key for Jenkins_int_rht
+  authorized_key:
+    user: root
```

Comment:
Why root, shouldn't jenkins connect as the jenkins user ?

Reply:
I'm guessing we need a `become`? Don't think we have the right keys to connect as root user directly